benchmarks/gogo/go_no_group/go_benchmark_test.go

```diff
@@ -0,0 +1,124 @@
+package main
```

Comment:
Can we use a script to create these variants from a common template?

Reply:
Done by add rule generate_all_gogo_benchmark_code